pages/common/xml-transform.md

```diff
@@ -0,0 +1,28 @@
+# xml-transform
+
+> Transform XML documents using XSLT.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- TBD:
+
+`xml transform`
+
+- TBD:
+
+`xml transform`
+
+- TBD:
+
+`xml transform`
+
+- TBD:
+
+`xml transform`
+
+- TBD:
+
+`xml transform`
```

Comment:
Again, this looks like a placeholder. As before, I recommend either:

1. Filling these out
2. Removing this page from this PR and potentially contributing it in a completed state in a future PR

Reply:
Again, please read the PR comment.